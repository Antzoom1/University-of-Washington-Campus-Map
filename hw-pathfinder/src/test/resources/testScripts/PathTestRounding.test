# Tests if the weights of the edges round
# and if the total cost also is rounded.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 7.335
AddEdge simpleGraph A C 3.047
AddEdge simpleGraph C D 2.658
AddEdge simpleGraph D B 9.124

FindPath simpleGraph A D