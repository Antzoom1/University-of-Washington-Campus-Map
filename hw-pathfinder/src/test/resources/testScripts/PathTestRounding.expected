# Tests if the weights of the edges round
# and if the total cost also is rounded.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 7.335 from A to B in simpleGraph
added edge 3.047 from A to C in simpleGraph
added edge 2.658 from C to D in simpleGraph
added edge 9.124 from D to B in simpleGraph

path from A to D:
A to C with weight 3.047
C to D with weight 2.658
total cost: 5.705